common/reviews/api/module-minifier-plugin.api.md

```diff
@@ -46,6 +47,7 @@ export const IDENTIFIER_TRAILING_DIGITS: string;
 
 // @public
 export interface IExtendedModule extends webpack.compilation.Module, webpack.Module {
+    external?: boolean;
```

Comment:
Should we call this `isExternal`?

Reply:
This already exists. It's used by some internal webpack plugins.